docs/features/autoupdater.rst

```diff
@@ -7,8 +7,11 @@ Building Images
 ---------------
 
 By default, the autoupdater is disabled (as it is usually not helpful to have unexpected updates
-during development), but it can be enabled by setting the variable GLUON_BRANCH when building
-to override the default branch set in the site configuration.
+during development), but it can be enabled by setting the variable ``GLUON_AUTOUPDATER_ENABLED`` to ``1`` when building.
+It is also possible to override the default branch during build using the variable ``GLUON_AUTOUPDATER_BRANCH``.
```

Comment:
I think this leaves out that setting `GLUON_AUTOUPDATER_BRANCH` implies `GLUON_AUTOUPDATER_ENABLED`.

Reply:
Oh, apparently only `GLUON_BRANCH` did that.

Reply:
Uhhh... it doesn't?